src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/model/request/PrisonerSearchByPrisonerNumbersRequest.kt

```diff
@@ -1,5 +1,5 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.model.request
 
 data class PrisonerSearchByPrisonerNumbersRequest(
-  val prisonerNumbers: List<String>,
+  val prisonerNumbers: List<String?>,
```

Comment:
mmmh... it might be better to filter out nulls if possible, rather than send them downstream